scripts/init.js

```diff
@@ -46,9 +48,14 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
   // Copy the files for the user
   fs.copySync(path.join(ownPath, 'template'), appPath);
 
-  // Rename gitignore after the fact to prevent npm from renaming it to .npmignore
-  // See: https://github.com/npm/npm/issues/1862
-  fs.move(path.join(appPath, 'gitignore'), path.join(appPath, '.gitignore'), []);
+  var gitignoreExists = pathExists.sync(path.join(appPath, '.gitignore'));
```

Comment:
This is why I originally wanted to only support the non existing folder case. Now we get in this situation where we have some files that already exist and need to be somehow merged. We can no longer be sure that all the configs that we have all work well together.

Could we check all the files that would be copied over and if any of them is already there then abort.

Honestly, i'm not sure it's worth the complexity, but I could be convinced otherwise